static/elements/chromedash-timeline.js

```diff
@@ -253,12 +263,12 @@ ORDER BY yyyymmdd DESC, client`;
 
   render() {
     return html`
-      <select .value="${this.selectedBucketId}" @change="${this.updateSelectedBucketId}">
-        <option disabled value="1">Select a property</option>
+      <input id="datalistinput" type="search" list="features" placeholder="Select or search a property" @change="${this.updateSelectedBucketId}" />
+      <datalist id="features">
         ${this.props.map((prop) => html`
-          <option value="${prop[0]}">${prop[1]}</option>
-          `)}
-      </select>
+          <option value="${prop[1]}" />
```

Comment:
This is good.  Not specifying a body to the option avoids the two-line display that we hit on a previous attempt to use datalist.

For debugging, I do like to be able to see the bucket IDs spelled out when I inspect this element.  So, please add a datalist-debug-bucket-id="${prop[0]}" here.

Reply:
Added the ```dataset-debug-bucket-id```